src/python/WMCore/WorkQueue/WorkQueue.py

```diff
@@ -139,7 +139,7 @@ def __init__(self, logger=None, dbi=None, **params):
         self.params.setdefault('WMBSUrl', None)  # this will only be set on local Queue
         if self.params.get('WMBSUrl'):
             self.params['WMBSUrl'] = Lexicon.sanitizeURL(self.params['WMBSUrl'])['url']
-        self.params.setdefault('Teams', [])
+        self.params.setdefault('Team', [])
```

Comment:
I would prefer this would be string not list. But I understand that would be many more code changes.

Reply:
I think I had a look at it today but decided not to change. I'll check that out tomorrow again, I believe it will be this file, the workRestrictions couch list and maybe WorkQueueBackend. I agree it will be much better, it's just more annoying to validate :)

Reply:
Actually it might not be that many change, (Maybe more then below).

This shouldn't be list
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WorkQueue/WorkQueueUtils.py#L133

And workRestrictions  
default string
https://github.com/dmwm/WMCore/pull/8464/files#diff-7e2ed5ba1c031d501f615be69a695be9R23
change to string comparison
https://github.com/dmwm/WMCore/pull/8464/files#diff-7e2ed5ba1c031d501f615be69a695be9R60

But I think in a long run it might be less confusing. Since current code implies Team indicates some times list, but some times string.